themes/default/content/blog/pulumi-release-notes-103/index.md

```diff
@@ -0,0 +1,105 @@
+
+{{< video title="Pulumi ESC editor enhancements demo" src="https://www.pulumi.com/uploads/esc-editor-enhancements-demo.mp4" autoplay="true" loop="true" >}}
+
+### Pulumi ESC OIDC customization 
+
+Pulumi ESC now offers [subject customization](/docs/pulumi-cloud/oidc/azure/#subject-customization) for OIDC tokens, enhancing security across AWS, GCP and Azure by aligning federated credentials with specific identifiers. By configuring the `subjectAttributes` setting, users can include specific environment, user, and organization information in the [OIDC](/docs/pulumi-cloud/oidc/) token subject claim, offering more granular control and customization for cloud resource access. This capability is especially valuable for customers using Azure, where subject claims are required to precisely match the string specified in the policy.
+
+### Pulumi ESC Audit Logs
+
+Pulumi ESC now features comprehensive Audit Logs for all Create, Read, Open, Update, and Delete (CRUD) operations, strengthening your cloud security framework. This critical addition to Pulumi ESC enables organizations to closely monitor and audit their cloud infrastructure changes, ensuring enhanced oversight and security compliance. Through detailed logging, teams gain valuable insights into their operational activities, facilitating improved governance and accountability across cloud resources.
+
+## Core
+
+### Dev Releases for the Pulumi CLI and SDKs
+
+We introduced [Dev Releases](/blog/introducing-dev-releases/), allowing developers to access the latest features and improvements in Pulumi's infrastructure as code tools ahead of their official release. With this initiative, we aim to engage our community and gather early feedback, enhancing the product development process. This is crucial for identifying and resolving potential issues early, contributing to a smoother transition to new releases. Dev Releases will also be at the cutting edge of Pulumi's evolving capabilities, with the latest bug fixes and features available within minutes of Pulumi developers merging. For instructions on how to use the CLI Dev releases see the [Install Docs](/docs/install/#installing-dev-releases), and for pre-release versions of Dev SDKs see the docs for [Node.js](/docs/languages-sdks/javascript/#dev-versions), [Python](/docs/languages-sdks/python/#dev-versions), [.NET](docs/languages-sdks/dotnet/#dev-versions), and [Go](/docs/languages-sdks/go/#dev-versions).
+
+### Automation API support to install Pulumi CLI 
+
+The Pulumi Automation API now offers the functionality to [install the Pulumi CLI](https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3/go/auto#InstallPulumiCommand) directly through the API. This feature enables deploying programs utilizing the Automation API without the need for prior CLI installation. Explore the [example code](https://github.com/pulumi/automation-api-examples/tree/main/go/cli_installation) to see how to integrate the new API seamlessly. 
```

Comment:
It's unfortunate we only have an example in Go. This is usable from the other languages that support automation API. Perhaps worth at least linking to the API docs for some of the other languages: 
- Node.js: https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/pulumi/classes/automation.PulumiCommand.html#install
- Python: https://www.pulumi.com/docs/reference/pkg/python/pulumi/#pulumi.automation.PulumiCommand.install
- .NET: Unfortunately, we need to manually kick off a doc update for the new API to show up in https://www.pulumi.com/docs/reference/pkg/dotnet/Pulumi.Automation/Pulumi.Automation.html. In the meantime, we could link directly to https://github.com/pulumi/pulumi-dotnet/blob/edf4a5e904110f198d579d8db0b178aa9af9cdfa/sdk/Pulumi.Automation/Commands/LocalPulumiCommand.cs#L107

Reply:
I added them